isamples_metadata/GEOMETransformer.py

```diff
@@ -2,15 +2,20 @@
 import datetime
 import logging
 import typing
+import re
 from typing import Optional
 
 import isamples_metadata
 from isamples_metadata.Transformer import (
     Transformer,
 )
 
+PERMIT_STRINGS_TO_IGNORE = ['nan', 'na', 'no data', 'unknown', 'none_required']
+
 TISSUE_ENTITY = "Tissue"
 JSON_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S%z"
+PERMIT_STRUCTURED_TEXT_COMPLIES_WITH_PATTERN = re.compile(r"complies_with:([^\s]+)(\s+authorized_by:(.*))?")
```

Comment:
Thinking about this a bit more- human entries are typically messy, especially during field work. Should probably also be prepared to accept tags like "compliesWith:", "complieswith:", maybe "complies with:", and perhaps even "complies:". Similarly for authorized_by.

Reply:
All of these should be working now -- I've tweaked the regexes and added tests.